sci-physics/vgm/vgm-4.5.ebuild

```diff
@@ -9,9 +9,10 @@ if [[ ${PV} == *9999* ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/vmc-project/${PN}.git"
 else
-	SRC_URI="http://ivana.home.cern.ch/ivana/${PN}.${PV}.tar.gz"
-	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
Why drop `*-linux` keywords?

Reply:
`sci-physics/root` does not have them in the recent ebuilds anymore, so I also have to drop them here (otherwise, `repoman` kills me since I have a revdep without these keywords ;-) ). 
If the dropping of these on `sci-physics/root` was not intentional, we can slo bring them back here.

Reply:
I don't recall dropping them on ROOT, which I do test on prefix-standalone at least, so we should add them back in root as well. You can drop for now, and I'll add them back when I test on prefix-guest.

Reply:
Ok, thanks! :-)